lib/administrate/generator_helpers.rb

```diff
@@ -4,10 +4,29 @@ def call_generator(generator, *args)
       Rails::Generators.invoke(generator, args, generator_options)
     end
 
+    def find_routes_file
+      routes_file = Rails.root.join("config/routes.rb")
+      if !File.file?(routes_file)
+        routes_file = find_file("routes.rb")
+        raise "Unable to locate your routes.rb file" unless !routes_file.nil?
```

Comment:
How do you think about doing this like:

`raise "…" if routes_file.nil?`

It saves us from checking for a negative negative (`unless not`).

Reply:
I think your suggested changes reads a lot nicer, I'll update that.